recipes/repository.rb

```diff
@@ -174,14 +178,18 @@ def warn_deprecated_yumrepo_gpgkey
     action :remove
   end
 
-  apt_repository 'datadog_apt_A2923DFF56EDA6E76E55E492D3A80E30382E94DE' do
+  apt_repository 'datadog_apt_5F1E256061D813B125E156E8E6266D4AC0962C7D' do
```

Comment:
No previous versions of the cookbook that were creating these `apt` repositories knew about this key, so we don't need to do this.

Reply:
So we add this during a next modification set when we remove old keys? I'm wondering if we could keep it if it's harmless?